### ISSUE TITLE
[feature request] a custom validation set

### ISSUE THREAD

**Issue by zygmuntz** [automl/auto-sklearn#147]
I would like to provide a custom validation set. As far as I can see, this is not possible in the current version.

**Comment by mfeurer** [comment]
This is not possible in the current version, but planned for the future. I'm closing this for now since it will be covered by the more general #149.
